### ISSUE TITLE
logs of push use inconsistent key names

### ISSUE THREAD

**Issue by elbidone** [meumobi/sitebuilder#154]
# What occurs
Current format is detailed below
```
[YYYY-MM-DD HH:MM:SS] sitebuilder.INFO: Sending push notification {"item id":"$","site id":$,"content":"$","devices":[]}
```
There's no reason to use "item id", instead of "item_id"
"site id" should be "site_id"
you can keep content if it's the name of key on payload sent. Else change it.
If the payload is displayed on debug mode, there's no need to show devices token twice. I suggest to only mention the number of device receivers.

# How to fix it
I suggest following format
```
[YYYY-MM-DD HH:MM:SS] sitebuilder.INFO: Sending push notification {"item_id":"...","site_id":...,"content":"...","number_of_devices":...}
```

where number_of_devices is an INT
should validate the key name "content" with the key name used on payload

**Comment by t4deu** [comment]
The key name is "content" according to [the API Doc](https://www.pushwoosh.com/programming-push-notification/pushwoosh-push-notification-remote-api/#PushserviceAPI-Method-messages-create) and the array returned by the "CreateMessageRequest::toJSON()" of the gomoob/php-pushwoosh lib:
```php
Array
(
    [application] => 1111-1111
    [applicationsGroup] => 
    [auth] => wez8slYDk....
    [notifications] => Array
        (
            [0] => Array
                (
                    [ignore_user_timezone] => 1
                    [send_date] => now
                    [content] => Item title
                    [ios_badges] => +1
                )
        )
)

```

**Comment by elbidone** [comment]
ok, and what about the others points ? yo only respond to question concerning "content" key name.

Should analyze how to resolve issue concerning
- site_id, item_id
- number_of_devices

**Comment by juliogreff** [comment]
This was resolved in #156, but the commit mentioned the pull request itself instead of the issue. I'm closing this myself.
